### PR TITLE
Pile-Up ID training and WPs for 2017 UL samples

### PR DESCRIPTION
**Pull request by marinakolosova**
#### PR description:

- With this PR new training files and working points for UL 2017 samples are added. New pt bins are introduced and the code changed accordingly. (Studies shown here [PU ID, UL17 slides](https://indico.cern.ch/event/916566/#44-ul17-pu-jet-id-update))
- There is also a PR for the new weights for PUJetID [PR#11](https://github.com/cms-data/RecoJets-JetProducers/pull/11)

#### PR validation:

Passed code-checks and code-format tests.
Passed runTheMatrix test workflows

FYI: @alefisico @camclean @singh-ramanpreet

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by alefisico** [comment]
Just to mention that these validation tests are going to fail because the PR needs these files: https://github.com/cms-data/RecoJets-JetProducers/pull/11

**Comment by slava77** [comment]
test parameters
- pull_request = cms-data/RecoJets-JetProducers#11

**Comment by slava77** [comment]
@cmsbuild please test

File: RecoJets/JetProducers/python/PileupJetIDCutParams_cfi.py [hunk 1]
```diff
@@ -46,19 +77,22 @@
     Pt010_Tight    = cms.vdouble( 0.26, -0.34, -0.24, -0.26),
     Pt1020_Tight   = cms.vdouble( 0.26, -0.34, -0.24, -0.26),
     Pt2030_Tight   = cms.vdouble( 0.26, -0.34, -0.24, -0.26),
-    Pt3050_Tight   = cms.vdouble( 0.62, -0.21, -0.07, -0.03),
+    Pt3040_Tight   = cms.vdouble( 0.62, -0.21, -0.07, -0.03),
+    Pt4050_Tight   = cms.vdouble( 0.62, -0.21, -0.07, -0.03),
```
Comment (slava77) [inline RecoJets/JetProducers/python/PileupJetIDCutParams_cfi.py:81]: are these WPs actually used?
I'm curious when is the opportunity to clean up this file.

Comment (alefisico) [reply RecoJets/JetProducers/python/PileupJetIDCutParams_cfi.py:81]: sorry you mean the 80X WPs or in general the Pt3040_* and Pt4050_*?

Comment (camclean) [reply RecoJets/JetProducers/python/PileupJetIDCutParams_cfi.py:81]: @slava77 the 80X WPs are indeed obsolete.

Comment (slava77) [reply RecoJets/JetProducers/python/PileupJetIDCutParams_cfi.py:81]: I meant 80X, 53X and alike.

Comment (camclean) [reply RecoJets/JetProducers/python/PileupJetIDCutParams_cfi.py:81]: However, we need this PR ASAP. We will make sure to clean up this file on the next iteration.

**Comment by slava77** [comment]
+1

for 
- code changes are in line with the PR description and linked slides (30-50 is split in two; the new IDs ```full_106x_UL17_chs``` are applied to all; the cuts in the slides are in agreement with changes in the code)
- jenkins tests pass and comparisons with the baseline show differences in the MVAPUID variables

data shows more clearly a drop in the fraction of passing jets
e.g. wf 136.874 (jetHT 2018; red is this PR)
wf136 874_ak4chs_loosePUID_pt
MC apparently has much less change, but the available tests done by the bot are done on just 10 events (compared to 100 for data).
wf250202 18_ak4chs_loosePUID_pt
These appear to be in line with the plots in the linked slides in the PR description.


cms-data/RecoJets-JetProducers#11 is needed when this PR is merged

**Comment by silviodonato** [comment]
+1

**Pull request merged**